w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml

```diff
@@ -38,128 +36,216 @@ filter_genes:
+    n_top_gene: ''
+    output_format: anndata_h5ad
+    span: '0.3'
 genes: {}
 gtf: {}
+harmony_batch:
+    adjusted_basis: X_pca
+    basis: X_pca
+    batch_key: BATCH_FIELD
+    input_format: anndata
+    output_format: anndata_h5ad
+    settings:
+        __current_case__: 0
+        default: 'true'
+make_project_file:
+    copy_adata_to_raw: 'false'
+    copy_e:
+        __current_case__: 0
+        default: 'true'
+        embedding_keys:
+        -   __index__: 0
+            contains: tsne
+        -   __index__: 1
+            contains: umap
+        embedding_sources:
+            __class__: ConnectedValue
+    copy_o:
+        __current_case__: 0
+        default: 'true'
+        obs_keys:
+        -   __index__: 0
+            contains: louvain
+        obs_sources:
+            __class__: ConnectedValue
+    copy_u:
+        __current_case__: 0
+        default: 'true'
+        uns_keys:
+        -   __index__: 0
+            contains: marker
+        uns_sources:
+            __class__: ConnectedValue
+    gene_flags: []
+    gene_symbols_field: index
+    modifications: []
+    output_format: anndata_h5ad
+    sanitize_varm: 'false'
+    top_genes: '50'
 matrix: {}
+merged_embeddings:
+    advanced:
+        conflict:
+            __current_case__: 3
+            duplicate_options: keep_first
+    inputs:
+    -   __index__: 0
+        input:
+            __class__: ConnectedValue
+    -   __index__: 1
+        input:
+            __class__: ConnectedValue
+merged_markers:
+    advanced:
+        conflict:
+            __current_case__: 3
+            duplicate_options: keep_first
+    inputs:
+    -   __index__: 0
+        input:
+            __class__: ConnectedValue
+    -   __index__: 1
+        input:
+            __class__: ConnectedValue
 n_neighbors:
     input_type:
         __current_case__: 0
         input_values: 3, 5, 10, 15, 20, 25, 30, 50, 100
         parameter_values: list_comma_separated_values
     parameter_name: n_neighbors
 neighbours:
-    __job_resource:
-        __current_case__: 0
-        __job_resource__select: 'no'
     input_format: anndata
-    output_format: anndata
+    output_format: anndata_h5ad
     settings:
         __current_case__: 1
         default: 'false'
+        key_added: ''
         knn: 'true'
         method: umap
-        n_neighbours: '15'
+        metric: euclidean
+        n_neighbors: '15'
+        n_neighbors_file:
+            __class__: RuntimeValue
         n_pcs: '50'
         random_seed: '0'
         use_rep: X_pca
 neighbours for umap:
     input_format: anndata
-    output_format: anndata
+    output_format: anndata_h5ad
     settings:
         __current_case__: 1
         default: 'false'
+        key_added: neighbors_n_neighbors_N_NEIGHBORS
         knn: 'true'
         method: umap
+        metric: euclidean
         n_neighbors: '15'
         n_neighbors_file:
-            __class__: RuntimeValue
+            __class__: ConnectedValue
```

Comment:
Sorry the `__class__` value is generated in an automated way when making parameter file using associated galaxy workflow or do we have control over it?

Reply:
This is entirely generated by the automated parameter generation script.